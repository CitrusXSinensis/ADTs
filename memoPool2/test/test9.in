ra1 0x
aa 1
aa 2
aa 5
aa 2
aa 1
aa 3
aa 1
aa 2
aa 5
aa 2
aa 1
aa 3
aa 1
aa 2
aa 5
aa 2
aa 1
aa 3
ca
pa
fa 1
fa 2
pa
fa 1
fa 4
fa 7
ca
pa
fa 3
pa
