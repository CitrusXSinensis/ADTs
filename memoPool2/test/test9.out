20/32
1 0 1 2 5 2 1 3 1 2 5 2 1 3 1 2 5 2 1 3
Chunk: 1
Chunk: 2
0 1 5 2 1 3 1 2 5 2 1 3 1 2 5 2 1 3
Chunk: 1
Chunk: 5
Nullptr
16/32
0 2 1 3 1 2 5 2 1 3 1 2 5 2 1 3
Chunk: 3
0 2 1 1 2 5 2 1 3 1 2 5 2 1 3
